RFC-0000-template.md

```diff
@@ -0,0 +1,57 @@
+# **RFC0 for Presto**
+
+See [CONTRIBUTING.md](CONTRIBUTING.md) for instructions on creating your RFC and the process surrounding it.
+
+## [Title]
+
+Proposers
+
+*
+*
+
+## [Related Issues]
+
+Related issues may include Github issues, PRs or other RFCs.
+
+## Summary
+
+Briefly describe what you intend to propose
+
+## Background
+
+Brief description of any existing issues, user requests or feature comparison with competitors which explains why the proposed feature might be needed. How the proposed feature helps our users? Explain the impact and value of this feature.
+
+### [Optional] Goals
+
+### [Optional] Non-goals
+
+## Proposed Implementation
+
+How do you intend to implement the feature? This section can be as detailed as possible with large subsections of its own, or may be a few sentences depending on the scope of the feature proposed. Explain the design in enough detail for existing users/contributors to understand. Design should include all the corner cases you can think of. Feel free to include any new SPI method signatures, class hierarchies or system contracts here. It is recommended to mention any methods, variables, classes which you think are needed to provide a broader view of the code changes. Please mention/describe the below on a high level -
```

Comment:
If the feature introduces SQL language changes or adds new system configuration, then we should describe these in this section. Right now these are in the adoption plan below. But I feel these user-interface changes should be described in the "Proposed implementation" as well. There might be feedback in terms of changing names or if they conflict with other better known usages of the new SQL etc.